app/src/main/java/com/portfolio/tasky/entry/fragments/LoginFragment.kt

```diff
@@ -42,7 +53,18 @@ class LoginFragment : Fragment(), FragmentInflater by FragmentInflaterImpl(), Te
 
         setObservers()
         viewBinding.btnLogin.setOnClickListener {
-            openRegistrationFragment()
+            val email = viewBinding.etEmail.subLayout.etInput.text.toString()
+            val password = viewBinding.etPassword.subLayout.etInput.text.toString()
+
+            lifecycleScope.launch(Dispatchers.IO + coroutineExceptionHandler){
+                viewModel.makeLoginCall(AuthenticationRequest(email, password))
```

Comment:
rename the method from "makeLoginCall" to something like "login"
Makes a bit more sense + you don't mention it's API related

Reply:
Done